docs/tutorial/geometry/index.rst

```diff
@@ -22,6 +22,7 @@ Geometry
     iss_keypoint_detector
     ray_casting
     distance_queries
+    uvmaps
```

Comment:
![Screenshot from 2023-08-12 18-14-35](https://github.com/isl-org/Open3D/assets/11634769/64ec5ae0-492b-48a0-b797-9b24d05cea9e)

In the pic, on left, there is a long heading - "Quick Reference to default UV Maps for some primitive shapes provided by Open3D", which I think should be shortened. 

Also, I chose Geometry -> Processing as the location to add UV Maps. Hopefully, its fine.

Reply:
Looks reasonable to me, thanks!